Automattic/jetpack
# packages: Fix some JS lints ahead of the eslint 9 upgrade

## Proposed changes:

To make the eslint 9 upgrade PR smaller, let's fix some of the lints that it'll complain about ahead of time.

### Other information:


## Jetpack product discussion


None

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* CI happy?
* Check the form export to CSV feature where some regexes were changed.
